application/src/main/data/upgrade/3.4.3/schema_update.sql

```diff
@@ -14,6 +14,16 @@
 -- limitations under the License.
 --
 
+
+-- ALARM ASSIGN TO USER START
+
+ALTER TABLE alarm ADD COLUMN IF NOT EXISTS assign_ts BIGINT;
+ALTER TABLE alarm ADD COLUMN IF NOT EXISTS assignee_id UUID;
+
+ALTER TABLE entity_alarm ADD COLUMN IF NOT EXISTS assignee_id UUID;
```

Comment:
The `entity_alarm` relation stores the relationship between the alarm and the propagated entity. I don't we need `assignee_id` there.